bqetl_project.yaml

```diff
@@ -20,6 +20,9 @@ default:
 
 dry_run:
   function: https://us-central1-moz-fx-data-shared-prod.cloudfunctions.net/bigquery-etl-dryrun
+  function_accounts:
+  - bigquery-etl-dryrun@moz-fx-data-shared-prod.iam.gserviceaccount.com
+  - bigquery-etl-dryrun@moz-fx-data-shar-nonprod-efed.iam.gserviceaccount.com
```

Comment:
It's possible to programatically get the service accounts from the url which is what I did first (in https://github.com/mozilla/bigquery-etl/pull/6288/commits/16fba8a72538e024b971bf4b362620f6d0d3a78a) but it doesn't seem worth the code complexity